library/src/androidTest/java/com/jmolsmobile/landscapevideocapture/configuration/CaptureConfigurationBuilderTest.java

```diff
@@ -1,12 +1,12 @@
 package com.jmolsmobile.landscapevideocapture.configuration;
 
-import android.support.test.runner.AndroidJUnit4;
-
 import com.jmolsmobile.landscapevideocapture.configuration.PredefinedCaptureConfigurations.CaptureResolution;
 
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import android.support.test.runner.AndroidJUnit4;
```

Comment:
Unneccessary import?

Reply:
I think it's on other " committing reformatting changes" error.